setup.py

```diff
@@ -15,7 +15,7 @@ def read_relative_file(filename):
 VERSION = read_relative_file('VERSION')
 PACKAGES = ['django_genericfilters']
 REQUIRES = [
-    'Django>=1.8,<1.12',
+    'Django>=1.8,<3.0',
```

Comment:
Why not support Django 3.0?

Reply:
2.2 is the LTS so I only needed this version. If 3.0 does not cause compatibility issues, why not :)

Reply:
Done :)